data/formats-data.js

```diff
@@ -3918,7 +3925,7 @@ exports.BattleFormatsData = {
 		tier: "PU",
 	},
 	stunky: {
-		randomBattleMoves: ["pursuit", "suckerpunch", "crunch", "fireblast", "explosion", "taunt", "playrough", "defog"],
+		randomBattleMoves: ["pursuit", "suckerpunch", "sludgebomb", "fireblast", "memento", "defog"],
```

Comment:
there's the super annoying thing about defog and play rough not legal on the same set, but play rough hits timburr/mienfoo which is important

Reply:
we dont care about such legalities but we're going to need broader benefit than that. its not like sludge does 0 to them anyway tho